### PR TITLE
fix: 'unsupport' diffusers, move inpainting models to compvis

### PR DESCRIPTION
**Pull request by tazlin**

### CODE REVIEW

**Comment by tazlin** [comment]
See ~~https://github.com/Haidra-Org/AI-Horde-Worker/pull/200~~ https://github.com/Haidra-Org/AI-Horde-Worker/pull/206 and https://github.com/Haidra-Org/AI-Horde-image-model-reference/pull/62 for more info,

**Comment by tazlin** [comment]
This will fail testing until https://github.com/Haidra-Org/AI-Horde-image-model-reference/pull/62 is merged.

File: tests/model_managers/test_hyper.py [hunk 1]
```diff
@@ -82,5 +88,12 @@ def test_get_mm_pointers(self):
         # Any value other than a string or a mm pointer is ignored
         assert SharedModelManager.manager.get_mm_pointers([None]) == set()
         assert SharedModelManager.manager.get_mm_pointers(
-            [None, SharedModelManager.manager.compvis, "FAKE", "esrgan", "compvis"],
+            [None, "FAKE", "esrgan", "compvis"],
```
Comment (db0) [inline tests/model_managers/test_hyper.py:91]: Why did you remove `SharedModelManager.manager.compvis` from the list? This is checking that the returned content is unique.

Comment (tazlin) [reply tests/model_managers/test_hyper.py:91]: I am a little uncertain as the the exact intention here, I guess. 

The function in question no longer accepts a reference, as I am a little unclear how you could get an instance of a model manager and then need to pass it to this function to get... the same reference back?

Comment (db0) [reply tests/model_managers/test_hyper.py:91]: Fair point, the original idea would be to handle everything, even when called internally. I don't remember what purpose I was considering when I made it handle pointers, perhaps except "handle everything"

Comment (tazlin) [reply tests/model_managers/test_hyper.py:91]: I think from a consumer point of view, I would like to try and encourage referring to the MMs by type or enum. See https://github.com/Haidra-Org/hordelib/pull/272/commits/12506961f95325b62dc5a62292506988cfe7b86a for a better picture on the function signatures (was wrong until that commit).

Comment (db0) [reply tests/model_managers/test_hyper.py:91]: I'm ok with enums, but they tend to require extra importing to handle. Anyway use your best judgement ;)

Comment (tazlin) [reply tests/model_managers/test_hyper.py:91]: It still accepts strings, which are castable to the Enum values (the enum in question is `class MODEL_REFERENCE_CATEGORIES(str, Enum):`), and from a trying to be pythonic viewpoint, I have no issue with allowing the primitive here.

Comment (db0) [reply tests/model_managers/test_hyper.py:91]: As always you show me the way it's properly done. :pray:

**Comment by tazlin** [comment]
Tox OK locally; confirmed worker is functioning with this version.
